cmd/notation/blob/policy/show.go

```diff
@@ -59,9 +59,9 @@ func runShow() error {
 		err = doc.Validate()
 	}
 	if err != nil {
-		fmt.Fprintf(os.Stderr, "Error: %s\n", err.Error())
-		fmt.Fprintf(os.Stderr, "Existing blob trust policy configuration is invalid, you may update or create a new one via `notation blob policy import <path-to-policy.json>`\n")
-		// not returning to show the invalid policy configuration
+		fmt.Fprintf(os.Stderr, "Existing blob trust policy file is invalid, you may update or create a new one via `notation blob policy import <path-to-policy.json>`. See https://github.com/notaryproject/specifications/blob/main/specs/trust-store-trust-policy.md#trust-policy-for-blobs for a blob trust policy example.\n")
+		os.Stdout.Write(policyJSON)
+		return err
```

Comment:
The behavior differs with the OCI policy show. Do you want to make them consistent?

Reply:
https://github.com/notaryproject/notation/pull/1126#discussion_r1899282139
As suggested by @Two-Hearts , OCI policy command related changes have been removed. Can we send another PR for update OCI policy command?